pages/policy.html

```diff
@@ -51,22 +45,32 @@ <h3>Cookies Policy</h3>
     <div class="section-item">
         <div class="content">
             <h3>About Cookies</h3>
-            <p>Cookies are information packets sent by web servers to web browsers, and stored by the web browsers.</p>
-            <p>The information is then sent back to the server each time the browser requests a page from the server.  This enables web servers to identify and track web browsers.</p>
-            <p>There are two main kinds of cookies: session cookies and persistent cookies.  Session cookies are deleted from your computer when you close your browser, whereas persistent cookies remain stored on your computer until deleted, or until they reach their expiry date.</p>
+            <p>
+                Cookies are small pieces of data stored in text files that are saved on your computer or other devices when websites are loaded in a browser.
+                They are widely used to remember you and your preferences, either for a single visit (through a “session cookie”) or for multiple repeat visits (using a “persistent cookie”).
```

Comment:
In response to [a previous review comment](https://github.com/OpenTTD/website/pull/252/files#r1186688389):
```suggestion
                They are widely used to remember you and your preferences, either for a single visit (through a "session cookie") or for multiple repeat visits (using a "persistent cookie").
```

Reply:
Oeh, you are sharp ... tnx :D